JSDemos/Demos/NumberBox/Overview/description.md

```diff
@@ -1 +1,8 @@
-The NumberBox is&nbsp;a&nbsp;UI component that displays a&nbsp;numeric value and allows a&nbsp;user to&nbsp;modify it&nbsp;by&nbsp;typing in&nbsp;a&nbsp;value, and incrementing or&nbsp;decrementing it&nbsp;using the keyboard or&nbsp;mouse.
\ No newline at end of file
+The NumberBox is a component that displays a number and allows users to modify it by typing in a value. Users can also increment or decrement the number using the keyboard arrows or mouse wheel.
+
+Use the [value](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#value) property to set the number displayed in the NumberBox. If the **value** property is not specified, the default number shown in the NumberBox is 0. You can also specify the [min](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#max) properties to set the value range.
+
+Set the [onValueChanged](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#onValueChanged) function to handle the value change. In this demo,  the value of the "Stock" NumberBox depends on the "This month sales" NumberBox. Change the value in the "This month sales" NumberBox to see how it affects the other one.
```

Comment:
In the demo, is it possible to change the "This month sales" NumberBox to "This month's sales"? If so, we can then update this paragraph as follows:

`Set the [onValueChanged](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#onValueChanged) function to handle the value change. In this demo, the value of the "Stock" NumberBox depends on the "This month's sales" NumberBox. Change the value in the "This month's sales" NumberBox to see how it affects the other value.`

Let me know what you think.

Reply:
I think it's not possible to change the name of this option since it's already named in the code of the demo this way.

Reply:
Okay, thanks for the clarification.